js/bitget.js

```diff
@@ -3627,6 +3627,105 @@ module.exports = class bitget extends Exchange {
         };
     }
 
+    async fetchFundingHistory (symbol, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitget#fetchFundingHistory
+         * @description fetch the funding history
```

Comment:
Would you like to add `@see https://bitgetlimited.github.io/apidoc/en/mix/#get-business-account-bill` in doc block?

Reply:
added